### PR TITLE
Adding more descriptive information to readme

### PR DESCRIPTION
**Pull request by jglev**
This follows a [discussion](https://github.com/greenelab/library-access/issues/14#issuecomment-362068564) in #14. A bit late in finishing it up, but I hope it will still be useful for readers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,31 @@ This repository is cataloging University Library access to scholarly literature.
...
+
+**The code files in this repository assume that your working directory is set to the top-level directory of this repository.**
+
+### Contents of this Repository, and the Order of Their Use
+
+- `LICENSE-*.md`: License text to accompany the [License](#License) section of this Readme below.
+- `environment.yml`: Conda environment file (see [Environment](#environment) below).
+- `.gitattributes`: File with information for tracking files using [Git Large File Storage (LFS)](https://git-lfs.github.com/).
+- `library_management_system_downloader` contains the following scripts, to be used in the following order:
+	- `downloader_configuration_file_TEMPLATE.py` should be copied to `downloader_configuration_file.py` and edited for your own institution's OpenURL resolver (These scripts were specifically tested using the OpenURL resolver that comes with Ex Libris' Alma management software).
```
Comment (dhimmel) [inline README.md]: For ordered things you may want to use a numbered list `1.`?

**Comment by dhimmel** [comment]
Yep. will be useful. Let me know when you're done adding commits on your end.

File: README.md (hunk 1 above)
Comment (jglev) [reply README.md]: Sure! Changed in b70710.

**Comment by jglev** [comment]
I think that this level of info. is enough to get others going. Do you agree?

**Comment by dhimmel** [comment]
> I think that this level of info. is enough to get others going. Do you agree?

It seems like the most important thing is undocumented -- what is `api_base_url`? How would someone go about getting one for their university?

**Comment by jglev** [comment]
Hmm, I can see how the documentation as it is could be confusing for a reader who's coming in from scratch to parse. This is a documentation difficulty; `api_base_url` is going to be very different from institution to institution, because it'll be based on whomever that institution contracts with for an OpenURL resolver. And finding out what that base URL is will be different for each institution, as it'll involve asking the local development team (e.g., in the library) for help with it / documentation for it. As we've talked about, it's also the case that different OpenURL resolvers may return slightly different formats of data.

Do you think it would be useful to state the above explicitly in the Readme? If so, do you think that that's sufficient for communicating to users?

**Comment by dhimmel** [comment]
> Do you think it would be useful to state the above explicitly in the Readme? If so, do you think that that's sufficient for communicating to users?

Yes.

**Comment by jglev** [comment]
Is 5f7d5a0 in line with what you had in mind, too?

**Review by dhimmel — APPROVED** [review]

**Comment by dhimmel** [comment]
Squash merged into https://github.com/greenelab/library-access/commit/310d718f8af0f2e519b84dc7ef72c660b858828e

**Pull request merged**